modules/sdk-coin-avaxp/src/avaxp.ts

```diff
@@ -90,6 +92,32 @@ export class AvaxP extends BaseCoin {
           }
         }
         break;
+      case TransactionType.Import:
```

Comment:
this is not import in C, this is import from c

Reply:
added check after this to confirm that we have import in c txn